packages/data-stores/src/site/types.ts

```diff
@@ -264,6 +264,7 @@ export interface Cart {
 	messages: Record< 'errors' | 'success', unknown >;
 }
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I don't feel so great about disabling the linter for this section. But, we didn't touch this code and it must be a new rule that's being enforced for typescript code?